regulations/static/regulations/js/source/views/comment/comment-view.js

```diff
@@ -96,7 +96,6 @@ var CommentView = Backbone.View.extend({
     this.model = blank ?
       new CommentModel(options) :
       comments.get(section) || new CommentModel(options);
-    this.listenTo(this.model, 'destroy', this.setSection.bind(this, section, tocId, label, true));
```

Comment:
I feel uneasy about this -- comment models are destroyed in multiple views. Removing this line means the `stopListening` above and the `render` below wouldn't be called and I'm not confident that that's safe.

Is it possible that this is a simple argument issue? It looks like the `indexes` parameter was added to `setSection` but not where it's being called here.